.github/workflows/notify-on-push.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         env:
           SLACK_COLOR: '#DE512A'
           SLACK_ICON: https://github.com/nodejs.png?size=48
-          SLACK_TITLE: Invalid commit was pushed to ${{ github.repository.default_branch }}
+          SLACK_TITLE: Invalid commit was pushed to ${{ github.repository.ref }}
```

Comment:
Shouldn't it be `github.ref`? Like the notifyOnForcePush job.

Reply:
Good catch
```suggestion
          SLACK_TITLE: Invalid commit was pushed to ${{ github.ref }}
```